#!/usr/bin/python3

def raise_exception_msg(message=""):
    """Send a message allong with a name error"""
    raise NameError(message)
